libebpfdiscoveryskel/src/SyscallProbes.h

```diff
@@ -210,12 +219,100 @@ __attribute__((always_inline)) inline static int handleSysRecvExit(struct pt_reg
 	return handleSysReadExit(ctx, bytesCount);
 }
 
+__attribute__((always_inline)) inline static int handleSysRecvmsgEntry(struct pt_regs* ctx, int fd, struct user_msghdr* msg, int flags) {
+	if (flags & MSG_PEEK) {
+		return 0;
+	}
+
+	if (flags & MSG_TRUNC || flags & MSG_OOB) {
```

Comment:
Please refactor this block of code for handling flags to a separate function as it's reused.